packages/core/src/app/Renderer.ts

```diff
@@ -80,6 +80,7 @@ export class Renderer {
         resolutionScale: 1,
         timeEventsClass: ReadOnlyTimeEvents,
       });
+      scene.variables.updateSignals(this.project.variables ?? {});
```

Comment:
I think we should do that in `reloadScenes` to stay consistent with the `Presenter` class.